### ISSUE TITLE
Light Phaser Master System crashes

### ISSUE THREAD

**Issue by Gusgus994** [TASEmulators/BizHawk#3595]
I wanted to play Gangster Town from Master System. I'm using BizHawk 2.8 and the core is SMSHawk. When I go to SMS and change the Sync Settings for any port into Light Phaser it prompts me to reboot core, after rebooting the following error message appear:

```
---------------------------
SMS load error
---------------------------
A core accepted the rom, but threw an exception while loading it:
System.InvalidOperationException: this AxisDict has been built and sealed and may not be mutated
   at BizHawk.Emulation.Common.AxisDict.AssertMutable()
   at BizHawk.Emulation.Cores.Sega.MasterSystem.SMSControllerDeck..ctor(SMSControllerTypes controller1, SMSControllerTypes controller2, Boolean is_GG, Boolean use_keyboard)
   at BizHawk.Emulation.Cores.Sega.MasterSystem.SMS..ctor(CoreComm comm, GameInfo game, Byte[] rom, SmsSettings settings, SmsSyncSettings syncSettings)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at BizHawk.Emulation.Cores.CoreInventory.Core.Create(ICoreInventoryParameters cip)
   at BizHawk.Client.Common.RomLoader.MakeCoreFromCoreInventory(CoreInventoryParameters cip, String forcedCoreName)
```

Image of it: https://imgur.com/a/RHiihp2

Then I can't load ANY Master System rom until I erase the config.ini file

Just casual player here. I'm using a Ryzen 5 3600, GeForce GTX 1650 and Windows 10 Pro N 22H2

Thanks

**Comment by YoshiRulz** [comment]
Duplicate of #3282. This is already fixed in the 2.9 release candidates.

**Comment by Gusgus994** [comment]
Thank you! Downloading the last 2.9 candidate rn. Sorry for the duplicate, guess I didn't search as well as I thought.
